.gitlab-ci.yml

```diff
@@ -162,16 +162,35 @@ variables:
   # Build images versions
   # To use images from datadog-agent-buildimages dev branches, set the corresponding
   # SUFFIX variable to _test_only
-  DATADOG_AGENT_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BUILDIMAGES: v32377621-ce42cb8e
-  DATADOG_AGENT_WINBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_WINBUILDIMAGES: v32377621-ce42cb8e
-  DATADOG_AGENT_ARMBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_ARMBUILDIMAGES: v32377621-ce42cb8e
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_SYSPROBE_BUILDIMAGES: v32377621-ce42cb8e
-  DATADOG_AGENT_BTF_GEN_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BTF_GEN_BUILDIMAGES: v32377621-ce42cb8e
+  CI_IMAGE_AGENT_DEPLOY_SUFFIX: ""
+  CI_IMAGE_AGENT_DEPLOY_VERSION: v32377621-ce42cb8e
+  CI_IMAGE_BTF_GEN_SUFFIX: ""
+  CI_IMAGE_BTF_GEN_VERSION: v32377621-ce42cb8e
+  CI_IMAGE_DDA_TESTING_SUFFIX: ""
+  CI_IMAGE_DDA_TESTING_VERSION: v32377621-ce42cb8e
+  CI_IMAGE_DEB_ARM64_SUFFIX: ""
+  CI_IMAGE_DEB_ARM64_VERSION: v32377621-ce42cb8e
+  CI_IMAGE_DEB_X64_SUFFIX: ""
+  CI_IMAGE_DEB_X64_VERSION: v32377621-ce42cb8e
+  CI_IMAGE_DOCKER_X64_SUFFIX: ""
+  CI_IMAGE_DOCKER_X64_VERSION: v32377621-ce42cb8e
+  CI_IMAGE_DOCKER_ARM64_SUFFIX: ""
+  CI_IMAGE_DOCKER_ARM64_VERSION: v32377621-ce42cb8e
+  CI_IMAGE_RPM_ARM64_SUFFIX: ""
+  CI_IMAGE_RPM_ARM64_VERSION: v32377621-ce42cb8e
+  CI_IMAGE_RPM_ARMHF_SUFFIX: ""
+  CI_IMAGE_RPM_ARMHF_VERSION: v32377621-ce42cb8e
+  CI_IMAGE_RPM_X64_SUFFIX: ""
+  CI_IMAGE_RPM_X64_VERSION: v32377621-ce42cb8e
+  CI_IMAGE_SUSE_X64_SUFFIX: ""
+  CI_IMAGE_SUSE_X64_VERSION: v32377621-ce42cb8e
+  CI_IMAGE_SYSPROBE_X64_SUFFIX: ""
+  CI_IMAGE_SYSPROBE_X64_VERSION: v32377621-ce42cb8e
+  CI_IMAGE_SYSPROBE_ARM64_SUFFIX: ""
+  CI_IMAGE_SYSPROBE_ARM64_VERSION: v32377621-ce42cb8e
+  CI_IMAGE_WINDOWS_SUFFIX: ""
+  CI_IMAGE_WINDOWS_VERSION: v32377621-ce42cb8e
```

Comment:
We have 2 windows images, `1809` and `ltsc2022`, shouldn't there be 2 tags for those?

Reply:
Like the debian arm images, they are build from the same definition today that's why I created a single variable for them